dev/system-test/firestore.ts

```diff
@@ -2265,6 +2280,40 @@ describe('QuerySnapshot class', () => {
   });
 });
 
+describe('BulkWriter class', () => {
+  let firestore: Firestore;
+  let randomCol: CollectionReference;
+
+  beforeEach(() => {
+    firestore = new Firestore({});
+    randomCol = getTestRoot(firestore);
+  });
+
+  afterEach(() => verifyInstance(firestore));
+
+  it.skip('increments and decrements the pendingOperationsCount', async () => {
+    const ref = randomCol.doc('doc1');
+    const writer = firestore._bulkWriter();
+    writer.set(ref, {foo: 'bar'});
+    writer.set(ref, {foo: 'bar2'});
+    await writer.close();
+    return firestore.terminate();
+  });
+
+  it('subsequent operations fail after terminate() is called', () => {
+    const ref = randomCol.doc('doc1');
+    const writer = firestore._bulkWriter();
+    return firestore.terminate().then(() => {
```

Comment:
Can you rewrite to follow this pattern: https://github.com/googleapis/nodejs-firestore/blob/master/dev/system-test/firestore.ts#L2060 ?

Reply:
done.

Reply:
I believe you forgot to push this, but maybe I am not supposed to review this yet :)

Reply:
I think you were viewing the outdated one?